packages/terra-doc-template/src/ExampleTemplate.jsx

```diff
@@ -1,12 +1,12 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import SyntaxHighlighter, { registerLanguage } from 'react-syntax-highlighter/dist/prism-light';
-import jsx from 'react-syntax-highlighter/dist/languages/prism/jsx';
+import SyntaxHighlighter from 'react-syntax-highlighter/dist/cjs/prism-light';
+import jsx from 'react-syntax-highlighter/dist/cjs/languages/prism/jsx';
 import classNames from 'classnames/bind';
 import styles from './ExampleTemplate.module.scss';
 import theme from './syntax-highlighter-theme';
 
-registerLanguage('jsx', jsx);
+SyntaxHighlighter.registerLanguage('jsx', jsx);
```

Comment:
Was this changed as a result of the new version of react-syntax-highlighter or just a preference?